transport/transport-runtime/src/main/java/com/google/android/datatransport/runtime/scheduling/jobscheduling/service/JobInfoSchedulerService.java

```diff
@@ -0,0 +1,76 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.android.datatransport.runtime.scheduling.jobscheduling.service;
```

Comment:
Package nesting seems excessive as well, for now lets just put everything in `com.google.android.datatransport.runtime.scheduling` and not create sub-packages.